package.json

```diff
@@ -19,18 +19,15 @@
     "jsii-pacmak": "^0.22.0",
     "jsii-rosetta": "^0.22.0",
     "lerna": "^3.20.2",
+    "standard-version": "^7.1.0",
     "typescript": "~3.8.2"
   },
   "repository": {
     "type": "git",
     "url": "git://github.com/aws/aws-cdk"
   },
   "standard-version": {
-    "releaseCommitMessageFormat": "v{{currentTag}}",
-    "scripts": {
-      "prebump": "echo $(node -pe \"require('./lerna.json').version\")",
-      "precommit": "git add ."
-    }
+    "releaseCommitMessageFormat": "v{{currentTag}}"
```

Comment:
I'd very much prefer we make it `release: v{{currentTag}}` (I am not dying on this hill however)

Reply:
Changed to `chore(release): v{{currentTag}}` to be compatible with conventional commits.